pmd-core/src/test/java/net/sourceforge/pmd/cli/ZipFileTest.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.cli;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import org.junit.Test;
+
+import net.sourceforge.pmd.PMDConfiguration;
+import net.sourceforge.pmd.PmdAnalysis;
+import net.sourceforge.pmd.lang.document.TextFile;
+
+public class ZipFileTest {
+
+    private static final String ZIP_PATH = "src/test/resources/net/sourceforge/pmd/cli/zipWithSources.zip";
+    private final Path zipPath = Paths.get(ZIP_PATH);
+
+    @Test
+    public void testZipFile() {
+        PMDConfiguration conf = new PMDConfiguration();
+        conf.addInputPath(zipPath);
+        try (PmdAnalysis pmd = PmdAnalysis.create(conf)) {
+            List<TextFile> files = pmd.files().getCollectedFiles();
+            assertThat(files, hasSize(3));
+            assertThat(files.get(0).getDisplayName(), equalTo(ZIP_PATH + "!/otherSrc/somefile.dummy"));
```

Comment:
Since there is no `relativizeRoot` configured, I would expect here an absolute path for the zip file.

It seems, currently we implicitly rely on the current working directory for both input path and the created display name...

Reply:
Actually reviewed the tests and kept the implementation as is.

Updated the documentation: If no relative-root-path is given, the reported path is the same like given as in `--dir`. I think, that's the least surprising behavior.

Added then back the sentence to use "/" to get absolute paths (we already had tests for this in ZipFileTest, added more tests in CoreCliTest)